service/src/main/java/bio/terra/workspace/service/resource/referenced/cloud/any/gitrepo/ReferencedGitRepoHandler.java

```diff
@@ -3,6 +3,9 @@
 import bio.terra.workspace.db.model.DbResource;
 import bio.terra.workspace.service.resource.model.WsmResource;
 import bio.terra.workspace.service.resource.model.WsmResourceHandler;
+import java.util.UUID;
+import javax.annotation.Nullable;
+import javax.ws.rs.BadRequestException;
```

Comment:
sorry for one last nit! this and the other referenced resources should use the Terra Common Library's ErrorReport class (so `bio.terra.common.exception.BadRequestException` instead of `javax.ws.rs.BadRequestException`)

Reply:
oh. My bad. I have not found this is default imported. Let me change it and merge it. Thank you for your help again. Zach.